BillTrack.Worker/Services/InvoicePdfUploader.cs

```diff
@@ -0,0 +1,30 @@
+using Amazon.S3;
+using Amazon.S3.Model;
+using BillTrack.Core.Interfaces.Repositories;
+using BillTrack.Core.Interfaces.Services;
+using BillTrack.Domain.Entities;
+
+namespace BillTrack.Worker.Services;
+
+public class InvoicePdfUploader : IFileUploader
+{
+    private readonly IAmazonS3 _s3Client;
+    
+    public InvoicePdfUploader(IAmazonS3 s3Client)
+    {
+        _s3Client = s3Client;
+    }
+
+    public async Task UploadFileToS3(Stream pdfStream, string bucketName, string fileName, Guid invoiceId)
```

Comment:
... when this method targets S3 and PDFs. Then it could be "UploadPdfToS3".
Is `Guid invoiceId` in use here?